frontend/src/services/clusterSettingsService.ts

```diff
@@ -12,7 +12,7 @@ export const fetchClusterSettings = (): Promise<ClusterSettings> => {
 };
 
 export const updateClusterSettings = (
```

Comment:
This might be out of scope for this PR, but as we are growing in functionality here, we might need to either refactor cluster settings to use the pass through API or at least refactor the API calls to the backend to make it similar to the k8s call we are performing, depending on the outcome of the conversation we might wanna track another issue as a follow up to these changes.

Reply:
hmm, I think you are right, I will mark this PR as WIP because I also want to remove it to pass-through API